### PR TITLE
Implement MyAccount download policy server API

### PR DESCRIPTION
**Pull request by IldarAbdullin-okta**
### Problem Analysis (Technical)
Not implemented

### Solution (Technical)
Prepare models and implement API for downloading policy

### Steps to reproduce:

Actual result:

Expected result:

### Tests
Added

### CODE REVIEW

**Review by FeiChen-okta — APPROVED** [review]

**Review by moisesolmedo-okta — APPROVED** [review]

File: Sources/DeviceAuthenticator/Networking/MethodSettingsModel.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    // TOTP generation algorithm
+    enum TOTPAlgorithms: String, Codable {
+        case HMACSHA1, HMACSHA256, HMACSHA512
+    }
+    // TOTP secret encoding format
+    enum TOTPSecretEncoding: String, Codable {
+        case Base32
+    }
+
+    enum TransactionType: String, Codable {
```
Comment (moisesolmedo-okta) [inline Sources/DeviceAuthenticator/Networking/MethodSettingsModel.swift:54]: did you add this? I named this model `TransactionTypesModel` to not conflict with the bitmask

**Review by moisesolmedo-okta — CHANGES_REQUESTED** [review]
Please see my comment about `TransactionType`

File: Sources/DeviceAuthenticator/Networking/MethodSettingsModel.swift (hunk 1 above)
Comment (IldarAbdullin-okta) [reply Sources/DeviceAuthenticator/Networking/MethodSettingsModel.swift:54]: Its full name is MethodSettingsModel. TransactionType, it is scoped by MethodSettingsModel namespace
I don't see any problem

Comment (moisesolmedo-okta) [reply Sources/DeviceAuthenticator/Networking/MethodSettingsModel.swift:54]: should be fine then.

**Pull request merged**